packages/p2p-v2/src/components/Modals/PreferredCountriesModal/PreferredCountriesFooter/PreferredCountriesFooter.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Button } from '@deriv-com/ui';
+
+type TPreferredCountriesFooterProps = {
+    isDisabled: boolean;
+    onClickApply: () => void;
+    onClickClear: () => void;
+};
+
+const PreferredCountriesFooter = ({ isDisabled, onClickApply, onClickClear }: TPreferredCountriesFooterProps) => {
+    return (
+        <div className='flex gap-[0.8rem] w-full'>
+            <Button
+                color='black'
+                disabled={isDisabled}
+                isFullWidth
+                onClick={onClickClear}
+                size='lg'
+                textSize='sm'
```

Comment:
the text sizes on responsive look a little smaller than design, can you check:

![Screenshot 2024-04-01 at 3 03 56 PM](https://github.com/binary-com/deriv-app/assets/103412909/2f6f60a7-3bdf-4797-815f-c9f488ffa3f7)

Reply:
Design file look:
![Screenshot 2024-04-01 at 3 04 58 PM](https://github.com/binary-com/deriv-app/assets/103412909/fad511c2-419b-4109-9633-04846831a709)